actions/list.action.ts

```diff
@@ -19,18 +19,18 @@ export class ListAction extends AbstractAction {
     await this.loadRegistry();
   }
   private async loadRegistry(): Promise<void>  {
-        const stencilFilePath = join(process.cwd(), '.stencil');
+        const stencilFilePath = join(process.cwd(), 'spec.yaml');
```

Comment:
don't hardcode the name, take this as input from the user, the user should be able to specify the path where the spec file is located.